guides/_subsections/download-edit-and-upload-a-customized-event-template.md

```diff
@@ -51,79 +51,79 @@ definition to download.
+        the target is not configured with <b>JMX</b> authentication then the
         connection attempt will continue without prompting for credentials.
       "
       text=navigate-to-events-include-text
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="Download the Template"
+      summary="Download the <i>Template</i>"
       image-name="2.3.0/download-edit-and-upload-a-customized-event-template-1.png"
       caption="
-        Click the action overflow three-dot menu on the right side of the
+        Click the action overflow \"&#65049;\" <i>three-dot</i> menu on the right side of the
         template entry in the table, then click <i>Download</i>.
       "
     %}
   </li>
   <li>
     {% capture edit-template-additional-content %}
-      The template definition is an XML file, so you may open and edit it
-      with your favourite text editor or XML document editor. The JMC
-      Template Manager can also be used for this purpose, as described
+      The template definition is an <code>XML</code> file, so you may open and edit it
+      with your favourite text editor or <code>XML</code> document editor. The <b>JMC
+      Template Manager</b> can also be used for this purpose, as described
       <a href="{{ page.url }}#edit-template-with-jmc">
         in this section
       </a>.
     {% endcapture %}
     {% include howto_step.html
-      summary="Edit the Template"
+      summary="Edit the <i>Template</i>"
       image-name="2.3.0/download-edit-and-upload-a-customized-event-template-2.png"
       caption=edit-template-additional-content
       text="
-        Edit the template definition to suit your requirements. When you are
-        satisfied with the new configuration, give your new template a
+        Edit the <i>Template</i> definition to suit your requirements. When you are
+        satisfied with the new configuration, give your new <i>Template</i> a
         meaningful and recognizable tag by editing the
         <code>&lt;configuration&gt;</code> element at the top of the file and
         setting the <code>label=</code> attribute to the desired name. You
         should also save the file according to this new name, but this is not
         strictly necessary. The <code>description</code> and
         <code>provider</code> attributes can also be used to help identify this
-        template later and to explain its goals and purpose.
+        <i>Template</i> later and to explain its goals and purpose.
       "
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="Open the Template upload dialog"
+      summary="Open the Template Upload Dialog"
       image-name="2.3.0/download-edit-and-upload-a-customized-event-template-3.png"
       caption="
-        Back on the Cryostat Events view, click the plus (<code>+</code>) icon
+        Back on the <i>Cryostat Events</i> view, click the plus (<code>+</code>) icon
```

Comment:
Wondering are we going to leave it the way it is as `+` or want me to change it to `upload` icon? on the UI it is more of an `upload` icon than `+` for this section?

Reply:
![image](https://github.com/cryostatio/cryostatio.github.io/assets/56743004/97ba8b4a-8fe4-4c4b-a97f-a7e8102f0883)